src/main/java/kernitus/plugin/OldCombatMechanics/utilities/Messenger.java

```diff
@@ -62,12 +62,18 @@ public static void sendWithPrefix(CommandSender sender, String message, String p
         send(sender, prefix + " " + message, args);
     }
 
-    public static void sendNormalMessage(CommandSender sender, String message, Object... args){
-        sendWithPrefix(sender, message, "&6[OCM]&r", args);
+    public static void sendNormalMessage(CommandSender sender, String message, Object... args) {
+        if (Config.getConfig().getBoolean("show-prefix", true))
```

Comment:
Is it not better to do `plugin.getConfig()` ?

Reply:
I guess. I was sort of just following the theme of using the utils provided and didn't see that the plugin was there.